model/team_member.go

```diff
@@ -69,6 +69,26 @@ func TeamsUnreadToJson(o []*TeamUnread) string {
 	}
 }
 
+func TeamUnreadToJson(o *TeamUnread) string {
+	b, err := json.Marshal(o)
+	if err != nil {
+		return ""
+	} else {
+		return string(b)
+	}
+}
+
+func TeamUnreadFromJson(data io.Reader) *TeamUnread {
+	decoder := json.NewDecoder(data)
+	var o TeamUnread
+	err := decoder.Decode(&o)
+	if err == nil {
+		return &o
+	} else {
+		return nil
+	}
+}
```

Comment:
Can you add tests for these two functions in `model/team_member_test.go`? There should be example tests very similar to these in other model test files